src/NodaTime/CalendarSystem.cs

```diff
@@ -938,22 +865,16 @@ private static class MiscellaneousCalendars
 
         private static class GregorianJulianCalendars
         {
-            internal static readonly CalendarSystem[] GregorianByMinWeekLength;
+            internal static readonly CalendarSystem Gregorian;
             internal static readonly CalendarSystem Julian;
 
             static GregorianJulianCalendars()
             {
                 var julianCalculator = new JulianYearMonthDayCalculator();
-                Julian = new CalendarSystem(CalendarOrdinal.Julian, JulianId, JulianName, new JulianYearMonthDayCalculator(), 4, new GJEraCalculator(julianCalculator));
+                Julian = new CalendarSystem(CalendarOrdinal.Julian, JulianId, JulianName, new JulianYearMonthDayCalculator(), new GJEraCalculator(julianCalculator));
```

Comment:
nit: appears to be unnecessarily creating a second `JulianYearMonthDayCalculator`?